.travis.yml

```diff
@@ -1,35 +1,73 @@
 language: rust
 rust: nightly
 sudo: false
+services: docker
+cache: cargo
```

Comment:
I've actually had pretty bad luck with this historicaly because the `target` directory is never cleaned up, is this required though?

Reply:
Done